src/client/src/app/site/[site]/view-availability/daily-appointments/daily-appointments-page.test.tsx

```diff
@@ -4,6 +4,11 @@ import { mockBookings } from '@testing/data';
 import { SearchParamsContext } from 'next/dist/shared/lib/hooks-client-context.shared-runtime';
 import { ReadonlyURLSearchParams } from 'next/navigation';
 
+const mockClinicalServices = [
+  { value: 'RSV:Adult', label: 'RSV (Adult)' },
```

Comment:
Doesn't invalidate the test or anything but we've recently changed `clinical_services.json` to have `RSV Adult` rather than `RSV (Adult)`. Might just be less confusing to use the correct service label here too